src/util/ContentTypes.ts

```diff
@@ -1,9 +1,10 @@
 // Well-known content types
-export const TEXT_TURTLE = 'text/turtle';
 export const APPLICATION_JSON = 'application/json';
 export const APPLICATION_OCTET_STREAM = 'application/octet-stream';
 export const APPLICATION_SPARQL_UPDATE = 'application/sparql-update';
 export const APPLICATION_X_WWW_FORM_URLENCODED = 'application/x-www-form-urlencoded';
+export const TEXT_HTML = 'text/html';
+export const TEXT_TURTLE = 'text/turtle';
```

Comment:
Surely we have that one? https://github.com/solid/community-server/pull/773/files#diff-c5d0c03936df6ad5144ca449ce867a50125f296f2a7ab2b87c7e2c83b4edd2d5R4

Reply:
I just changed the order so they're alphabetical